.github/main.yml

```diff
@@ -0,0 +1,27 @@
+name: Build and run tests
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
```

Comment:
```suggestion
      uses: actions/checkout@v4
```

Reply:
These libs are at version 4 and though the core functionality should be the same, they do address bugs from earlier ones.